package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "lerna run lint",
     "cs-check": "lerna run cs-check",
     "cs-format": "lerna run cs-format",
-    "build": "lerna run --stream build",
+    "build": "lerna run --concurrency 1 --stream build",
```

Comment:
@ranihorev I was serious when I suggested adding this in as `build-serial` and documenting it in the `contributing.md`... If you do, please restore the blank line at the end of the file

Reply:
I'll do that. I'm still trying to figure out if this fixes Netlify. There is no error message so it's mainly trial and error 🤷

Reply:
Yes, it fixes it. I'll add `build-serial`

Reply:
I don't think that we need to update the doc actually. `build-serial` will only run on Netlify which is not mentioned in that doc. wdyt?

Reply:
I think it might be helpful to let developers who have under-powered machines know that they can run `npm run build-serial` if they are not successfully running `npm run build`